tests/test_createpythonpackage.py

```diff
@@ -40,6 +40,7 @@ def test_defaults(self):
     def run_package(self, config: Dict):
         with tempfile.TemporaryDirectory(prefix="cpp_tests_") as t:
             config["path"] = Path(t)
+            config["package_name"] = "testpackage"
```

Comment:
What do you think of using `test-package` to exercise the new code?

Reply:
Yep, absolutely. Actually I still have to add proper tests for this!

Reply:
No rush, but are you still planning on finishing this PR? LMK if you want me to take it over.

Reply:
Hey, sorry to have let this behind a bit. Not much time for open source at the moment unfortunately, so I need to prioritize. You can take it over if you're up for it. :-) Just check out and copy my branch to your local!